test/index.test.js

```diff
@@ -0,0 +1,96 @@
+import { describe, beforeEach, expect, it, vi, afterEach } from 'vitest'
+import { checkBackend } from '../src/index'
+import * as nodeFetch from 'node-fetch'
+
+vi.mock('node-fetch', async () => {
+  const actual = await vi.importActual('node-fetch')
+  return {
+    ...actual,
+    default: vi.fn(),
+  }
+})
+const fetch = vi.mocked(nodeFetch.default)
+// const mockedCheckBackend = vi.fn(checkBackend)
+// const mockSleep = vi.fn(sleep)
+
+describe('test index', () => {
+  beforeEach(() => {
+    vi.useFakeTimers()
+  })
+
+  afterEach(() => {
+    vi.restoreAllMocks()
+  })
+
+  it('should return up if backend is up', async function () {
+    const version = '1.0.0'
```

Comment:
Maybe nitpicking here, but the version is a short commit hash, not a semantic version